galois_gen_amd64.s

```diff
@@ -1,6 +1,6 @@
 // Code generated by command: go run gen.go -out ../galois_gen_amd64.s -stubs ../galois_gen_amd64.go -pkg=reedsolomon. DO NOT EDIT.
 
-//go:build !appengine && !noasm && !nogen && gc
+// go:build !appengine && !noasm && !nogen && gc
```

Comment:
Is this a valid change? I though only `//go:build` (without a space) was valid.

Reply:
Not sure. I have to manually add it, since avo doesn't add it when run with Go 1.18, but older versions expect it. Too much magic.

Removing it.